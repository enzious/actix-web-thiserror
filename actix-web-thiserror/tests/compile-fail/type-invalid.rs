use actix_web_thiserror::ResponseError;
use thiserror::Error;

#[derive(Debug, Error, ResponseError)]
enum MacroErrors {
  #[response(type = 10)]
  #[error("invalid type")]
  InvalidType,
}

fn main() {}
