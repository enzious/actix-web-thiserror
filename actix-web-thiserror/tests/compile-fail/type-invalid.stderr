error: proc-macro derive panicked
 --> tests/compile-fail/type-invalid.rs:4:24
  |
4 | #[derive(Debug, Error, ResponseError)]
  |                        ^^^^^^^^^^^^^
  |
  = help: message: Invalid `type` in #[response]
